crates/walrus-stress/src/generator/write_client.rs

```diff
@@ -78,16 +76,23 @@ impl WriteClient {
 
     /// Stores a fresh consistent blob and returns the blob id and elapsed time.
     pub async fn write_fresh_blob(&mut self) -> Result<(BlobId, Duration), ClientError> {
-        self.write_fresh_blob_with_epochs(1).await
+        self.write_fresh_blob_with_epochs(None).await
     }
 
-    /// Stores a fresh blob with the given number of epochs ahead, and returns the blob id and
-    /// elapsed time.
+    /// Stores a fresh blob and returns the blob id and elapsed time.
+    ///
+    /// If `epochs_to_store` is not provided, the blob will be stored for the number of epochs
+    /// randomly chosen between `min_epoch_to_store` and `max_epoch_to_store` specified in the
+    /// blob config.
     pub async fn write_fresh_blob_with_epochs(
         &mut self,
-        epochs_to_store: EpochCount,
+        epochs_to_store: Option<EpochCount>,
```

Comment:
```suggestion
```
Looks like we can just remove this parameter, and maybe rename this function to `write_fresh_blob`.

Reply:
This arg currently is still used in initializing a blob for reads, where we specify an epoch.